password.py

```diff
@@ -1,49 +1,302 @@
 # Password Creator/Manager
 
+import random
+import shane_char 
+import re
+
+account_dict = {}
+account_info_dict = {}
 
 #<---- FIRST MAJOR FUNCTION ---->
 # Creates an account
 def account_create():
 
-    return
+    print("-"*18, "\nAccount Creation!"), print("-"*18)
+
+    while True:
+        username = input("Enter a Username:\n")
+        if len(username) < 5:
+            print("Username has to be 5 characters or more.")
+            continue
+        else:
+            account_info_dict["Username"] = username
+            break
+
+    if password_randomizer_query() == 'no':
+        while True:
+            password = input("Enter a Password:\n")
+            if len(password) < 6:
+                print("Password has to be 6 characters or more.")
+                continue
+            else:
+                account_info_dict["Password"] = password
+                break
+    
+    account_dict[username] = account_info_dict
+
+    security_questions()
+
+    account_save()
+
+    return 
+
+# Asks user to pick a question and saves the answer
+def security_questions():
```

Comment:
I think I said this somewhere else, but when you create functions try to name them after what they're doing in action form. So the first word of your function name should usually be an action like: "generate, print, save, list, etc".

This function should instead be `get_security_questions` or `handle_security_questions`

You should generally do this for most functions except for main.